dagql.ebnf

```diff
@@ -12,6 +12,7 @@ cond = "TRUE"
      | "FALSE"
      | expr (">" | "<" | ">=" | "<=" | "=" | "<>") expr
      | "NOT" cond
+     | cond ("AND" | "OR") cond
```

Comment:
Shouldn't conditions support parentheses to specify order of operations? *e.g.* `a AND (b OR c)` vs `(a AND b) OR c`.

Reply:
Yup that's an oversight on my part. I'll open a new PR this evening to clean this up.